docs/zkapps/o1js/smart-contracts.md

```diff
@@ -166,7 +131,7 @@ As a zkApp developer, if you add this method to your smart contract, you are say
 
 ## Reading state
 
-Often, we also want to _read_ state – check out this example:
+This example _reads_ state:
```

Comment:
do we want to provide a reason to read state? (the often statement can be enhanced)

Reply:
Maybe leave it as is until we start getting questions about this?